src/app/store/slices/storage/storage.thunks/uploadFolderThunk.ts

```diff
@@ -11,10 +11,14 @@ import tasksService from '../../../../tasks/services/tasks.service';
 import { TaskStatus, TaskType, UploadFolderTask } from '../../../../tasks/types';
 import { planThunks } from '../../plan';
 import workspacesSelectors from '../../workspaces/workspaces.selectors';
+
+import { checkFolderDuplicated } from '../folderUtils/checkFolderDuplicated';
+import { getUniqueFolderName } from '../folderUtils/getUniqueFolderName';
 import { StorageState } from '../storage.model';
 import { deleteItemsThunk } from './deleteItemsThunk';
 import { uploadItemsParallelThunk } from './uploadItemsThunk';
 
+// TODO: REMOVE IROOT from this file, it is in types.ts
```

Comment:
Can you delete it, or is it imported in a pile of files?

Reply:
Right, it is used in a lot of files and I didn't want to increase the size of the changes and make it difficult to review the PR. We will do it in a separate PR so that you can more easily see the changes with respect to this